numpy/distutils/system_info.py

```diff
@@ -540,7 +540,7 @@ def __init__(self,
                     'extra_compile_args': '', 'extra_link_args': ''}
         self.cp = ConfigParser(defaults)
         self.files = []
-        self.files.extend(get_standard_file('.numpy-site.cfg'))
+        self.files.extend(get_standard_file('~/.numpy-site.cfg'))
```

Comment:
Interesting that we missed this for so long. 

This change is not cross-platform, should be `os.path.join(os.environ['HOME'], 'numpy-site.cfg')`